src/lib/y2network/widgets/vlan_id.rb

```diff
@@ -0,0 +1,37 @@
+require "cwm/common_widgets"
+
+module Y2Network
+  module Widgets
+    class VlanID < CWM::IntField
+      def initialize(config)
+        textdomain "network"
+
+        @config = config
+      end
+
+      def label
+        _("VLAN ID")
+      end
+
+      def help
+        # TODO: previously not exist, so write it
+      end
+
+      def init
+        self.value = (@config["VLAN_ID"] || "0").to_i
```

Comment:
It seems to not be based on the proposed name anymore

Reply:
yes, it is true as name is not defined before vlan id anymore.

Reply:
Yep, but at least would be nice to be the same than the proposed name. So currently if you have more than one in order starting with 0... the ID will be always 0. But that is something not critical, just something nice to have.